### ISSUE TITLE
How to set HTML parameters (e.g. disable_git, edit_link)?

### ISSUE THREAD

**Issue by roland-KA** [JuliaDocs/Documenter.jl#2017]
It might be a trivial issue, but I don't know how to do it:

The documentation says that keyword arguments like `disable_git`or `edit_link` can be set for `HTMLWriter` on a call to `makedocs`. But how? What is the exact syntax? An example would be nice.

**Comment by mortenpi** [comment]
Yeah, I think we could do better with the docs here. In short, you need to pass them via `format = Documenter.HTML(...)`: https://documenter.juliadocs.org/stable/lib/internals/writers/#Documenter.Writers.HTMLWriter.HTML

**Comment by roland-KA** [comment]
Ah thanks for the fast response! That works. 😊

I've tried `Documenter` on a local repo which is not (yet) under Git control. Therefore `Documenter` complained about "_Unable to determine HTML(edit_link = ...)_ ...". 

Setting `edit_link = nothing` avoids this error message, but I'm surprised that `disable_git` doesn't have that effect.

**Comment by mortenpi** [comment]
> Therefore Documenter complained about "Unable to determine HTML(edit_link = ...) ...".

Is that with 0.27.23 or master? There have been some changes to how we handle Git links on master that will be in 0.28.0.

**Comment by roland-KA** [comment]
I'm currently using 0.27.23
